framework/src/bundle/BundleRegistry.cpp

```diff
@@ -43,6 +43,17 @@
 
 namespace cppmicroservices {
 
+// Helper class to ensure RAII for InitialBundleMap in case where Install0 throws
+class InitialBundleMapCleanup {
+public:
+  InitialBundleMapCleanup(std::function<void()> cleanupFcn) : _cleanupFcn(cleanupFcn) {}
```

Comment:
use ```std::move(cleanupFcn)``` in the constructor's member initializer list since the lambdas you pass in aren't called by anyone else.

Reply:
Added std::move per Jeff's suggestion